### PR TITLE
Give Robotics medical modules an extinguisher

### PR DESCRIPTION
**Pull request by GoldenKeyboard**
People were complaining about the medical module not having an extinguisher.
And Mark didn't want to do it :(

### CODE REVIEW

**Comment by Tarmunora** [comment]
There are some good arguments against this in the [forum thread](https://forum.ss13.co/showthread.php?tid=14614) - notably regarding specialization and cooperation

**Comment by mordent-goonstation** [comment]
This PR is obviously technically sound, but I have objections based on the thread linked by @Tarmunora. I'm not entirely against it, but think it makes the medical module even _more_ appealing than it already is.

**Comment by GoldenKeyboard** [comment]
no no no, I'm testing things, fucking sync command

**Comment by Flaborized** [comment]
I think it's more interesting and fun for mediborgs to not have extinguishers. I prefer them having to use scalpels (or the new more dangerous circular saw!!) as a weapon and I think that fits the aesthetic of medical cyborgs attacking things. I also think it's funny how they have to use burn menders on burning patients. Just because something would be useful for a job doesn't mean it's a good idea imo

**Comment by zjdtmkhzt** [comment]
Just to make an argument in favour of this PR, I really feel like "it's funny to have medborgs drag burning bodies around" and "it forces cooperation" are really bad arguments here.
Borgs are supposed to be good at what they do, the primary use of a fire extinguisher is to extinguish fires, most of the time these are fires on humans. It makes zero sense to me for the module that has the responsibility to watch over human health to not have the tool that is normally used for extinguishing humans.
Of course, a borg having to drag a human around is kind of funny, but none of the other modules have things like these (besides the Brobot, which is just overall intended as a joke module).
_All_ the other serious modules have an extinguisher even though the medical one would undoubtedly need it the most, which is just completely silly.
Borgs should not have to sacrifice one of their three beaker slots to carry water around to extinguish burning people, this makes less sense than just having an extinguisher while at the same time also being unintuitive for new players, which I really think is the worst thing about it.

TLDR: I think that we shouldn't deny medical borgs a tool that they really should have for the reason that "it's kind of funny". We mostly try to give all the other modules the tools they need to do their jobs well and using more obscure methods to extinguish burning organics is something that may not occur to newer borg players (and something that I feel should not be necessary).
If the medical module is picked too often, that is a balance concern that should be adressed in some other way that does not run contrary to common sense on what kinds of equipment borgs should have for their jobs.
Lastly, medical borgs having access to a strong melee weapon is not really a big factor, seeing as they have access to the new, more dangerous surgery tools. Even if some opt to use the extinguisher instead, surgery tools are still a strong option and I think that most will keep using them. (Especially if they have the ability to mix poisons.)
This TLDR is now about as long as the section before it, but I really think medical borgs should get extinguishers, so I don't care!

**Comment by Tiggersaurus** [comment]
Thank you for putting into words what I was trying to, brain go brrr.

If people are *that* focused on the PvP aspect rather than the helping save lives/helping put out medbay plasma fires then why not either do a pocket fire extinguisher (recoloured smol blue o2 tank maybe?) or do similar to Borg botany saw/deconstructor and reduce the damage?

I just want my humans to not be on fire :(

**Comment by Flaborized** [comment]
IMO using the burn menders to deal with being on fire *is* the mediborgs tool for burning people. I think this is cool and I also think that it opens up situations where mediborgs have to prioritize who is burning to death more intensely when there are big fire explosions. I think this is more interesting, trying to juggle multiple patients who are burning to death, than it is to just have the tool to put them all out. It also opens up unique answers to burning patients, like dragging them through puddles of water.

**Comment by zjdtmkhzt** [comment]
Hmm, I feel like just burn mending people or dragging them through puddles is really unimmersive though. As we all know, this game is incredibly realistic in all other aspects, and realistically speaking it would not be effective to drag a person through a shallow puddle or to heal their burn wounds as they are acquiring them.
In addition, does it not strike you as strange that the module that is specifically made to care for humans is the worst at it in this particular situation?

While it might lead to fun improvisation sometimes, I feel like in the majority of cases it just leads to people burning to death. I feel as though there are plenty of places for fun improvised treatments in this game, but in places where it feels less ... silly. And I mean silly from a game design perspective, not from a logic perspective.

**Comment by mordent-goonstation** [comment]
Could we concede "leave this as-is for now" the caveat that I'm attending to solve this problem a different way once we have TGUI? (No point doing it sooner because it's a good amount of work for a relatively short release time, in theory)

**Comment by Tiggersaurus** [comment]
Given the other QoL changes I've seen you do for Borg, I trust and look forward to seeing what you come up with 💜

**Comment by GoldenKeyboard** [comment]
Mordant intends to solve this at a later date in a "different  way"

**Comment by UrsulaMejor** [comment]
Imo the medical borgs should get an extinguisher, if for the only reason that they are currently the one and only borg module that can get permanently stuck in space

**Comment by Tiggersaurus** [comment]
I thought to comment on that but whenever I've said before I got the "hurrhurr just use propulsion upgrade nerd" reply.
